ci/glci/azure.py

```diff
@@ -162,6 +175,51 @@ def publish_offer(self, publisher_id: str, offer_id: str, notification_mails):
             )
         response.close()
 
+    def fetch_ongoing_operation_id(self, publisher_id: str, offer_id: str, transport_dest: Azmp_transport_dest):
+        '''Fetches the id of an ongoing Azure Marketplace transport operation to a certain transport destination.'''
+
+        response = requests.get(
+            f"https://cloudpartner.azure.com/api/publishers/{publisher_id}/offers/{offer_id}/submissions",
```

Comment:
could you factor out the api-routes (deduplicate base URL, separate URL construction from code using them)? e.g. like so: https://github.com/gardener/cc-utils/blob/master/checkmarx/client.py#L30 (there are probably also other patterns one could apply, of course)